.github/ISSUE_TEMPLATE/documentation_issue.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Documentation issues
+about: Report issues with the Microsoft Edge documentation content.
```

Comment:
```suggestion
about: Report issues with the Microsoft Edge documentation content.
```
```suggestion
about: Report issues with the Microsoft Edge documentation.
```
Now that the design has a single template for docs issues, no longer need the distinction between documentation "bugs" vs. documentation "content".  In this revised design, in the phrase "documentation content", adding the word 'content' might actually reduce clarity.